src/applications/disability-benefits/526EZ/components/DisabilityWizard.jsx

```diff
@@ -0,0 +1,230 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import appendQuery from 'append-query';
+import classNames from 'classnames';
+
+import ErrorableCheckboxes from './ErrorableCheckboxes';
+import ErrorableRadioButtons from '@department-of-veterans-affairs/formation/ErrorableRadioButtons';
+
+import { toggleLoginModal } from '../../../../platform/site-wide/user-nav/actions';
+
+import ButtonContainer from './ButtonContainer';
+import { GetStartedMessage, disabilityStatusOptions, disabilityUpdateOptions, layouts } from '../helpers';
+
+const { chooseStatus, chooseUpdate, applyGuidance } = layouts;
+
+class DisabilityWizard extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentLayout: chooseStatus
+    };
+  }
+
+  getDisabilityStatus = () => {
```

Comment:
Is this only returning `appeal`, `first`, `undefined`, or `update` because of the first question? If so, can you put a comment describing why this function does this?

Reply:
Good idea, comment added and renamed.